cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -421,11 +436,12 @@ $('button[value="add-new-payment"]').on('click', function (e) {
 });
 
 function setPaymentData() {
-    $('#adyenEncryptedCardNumber').val(card.paymentData.encryptedCardNumber);
-    $('#adyenEncryptedExpiryMonth').val(card.paymentData.encryptedExpiryMonth);
-    $('#adyenEncryptedExpiryYear').val(card.paymentData.encryptedExpiryYear);
-    $('#adyenEncryptedSecurityCode').val(card.paymentData.encryptedSecurityCode);
-    $('#cardOwner').val(card.paymentData.holderName);
+    $('#adyenEncryptedCardNumber').val(card.state.data.encryptedCardNumber);
+    $('#adyenEncryptedExpiryMonth').val(card.state.data.encryptedExpiryMonth);
+    $('#adyenEncryptedExpiryYear').val(card.state.data.encryptedExpiryYear);
+    $('#adyenEncryptedSecurityCode').val(card.state.data.encryptedSecurityCode);
```

Comment:
should these values be "card.state.data.paymentMethod.encryptedSecurityCode" like here? https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/168/files#diff-81c46487cee32a4e6ebac5e05d313f83R88